README.md

```diff
@@ -116,180 +113,170 @@ func main() {
-for use with [Confluent Cloud](https://www.confluent.io/confluent-cloud/).
-
+More elaborate examples are available in the [examples](examples) directory, including
+[how to configure](examples/confluent_cloud_example) the Go client for use with
+[Confluent Cloud](https://www.confluent.io/confluent-cloud/).
 
-Getting Started
-===============
+## Installation
 
-Installing librdkafka
----------------------
+### Installing librdkafka
 
-This client for Go depends on librdkafka v1.1.0 or later, so you either need to install librdkafka
-through your OS/distributions package manager, or download and build it from source.
+confluent-kafka-go depends on [librdkafka](https://github.com/edenhill/librdkafka) v1.1.0 or later.
+See [librdkafka/README/Installation](https://github.com/edenhill/librdkafka#installation) for
+installation instructions.
 
-- For Debian and Ubuntu based distros, install `librdkafka-dev` from the standard
-repositories or using [Confluent's Deb repository](http://docs.confluent.io/current/installation.html#installation-apt).
-- For Redhat based distros, install `librdkafka-devel` using [Confluent's YUM repository](http://docs.confluent.io/current/installation.html#rpm-packages-via-yum).
-- For MacOS X, install `librdkafka` from Homebrew. You may also need to brew install pkg-config if you don't already have it. `brew install librdkafka pkg-config`.
-- For Alpine: `apk add librdkafka-dev pkgconf`
-- confluent-kafka-go is not supported on Windows.
+### Installing confluent-kafka-go
 
-Build from source:
-
-    git clone https://github.com/edenhill/librdkafka.git
-    cd librdkafka
-    ./configure --prefix /usr
-    make
-    sudo make install
+#### Latest stable release
 
+Install the package from [gopkg.in](https://gopkg.in).
+```bash
+go get -u gopkg.in/confluentinc/confluent-kafka-go.v1/kafka
+```
 
-Install the client
--------------------
+Import the `kafka` package in your code:
+```golang
+import "gopkg.in/confluentinc/confluent-kafka-go/kafka"
+```
 
-We recommend that you version pin the confluent-kafka-go import to v1:
+#### Master
 
-Manual install:
+Install the package from GitHub.
 ```bash
-go get -u gopkg.in/confluentinc/confluent-kafka-go.v1/kafka
+go get -u github.com/confluentinc/confluent-kafka-go/kafka
 ```
 
-Golang import:
+Import the `kafka` package in your code:
 ```golang
-import "gopkg.in/confluentinc/confluent-kafka-go.v1/kafka"
+import "github.com/confluentinc/confluent-kafka-go/kafka"
 ```
 
-**Note:** that the development of librdkafka and the Go client are kept in synch.
-If you use the master branch of the Go client, then you need to use the master branch of
-librdkafka.
+**Note**: the development of confluent-kafka-go and librdkafka are kept in sync. If you install
+confluent-kafka-go from master, then you will also need to install librdkafka from master.
 
-See the [examples](examples) for usage details.
+#### Using Go 1.13+ Modules
```

Comment:
This chapter is good, I recommend reverting all the versioning changes above and just adding this chapter.